doc/manual/manual.tex

```diff
@@ -307,7 +307,7 @@ \subsection{Referencing \aspect{}}
   pages =        {12--29}
 }
 \end{lstlisting}
-Please also cite the User's Manual:
+cite the User's Manual:
```

Comment:
Or did you mean to say something like "You can also cite the manual as follows:"